packages/core/src/Stores/client-store.js

```diff
@@ -287,6 +287,12 @@ export default class ClientStore extends BaseStore {
 
     @computed
     get is_logged_in() {
+        console.log(!!(
```

Comment:
@mustofa-binary do you need the console.log here?

Reply:
thats the eslint error im removing 😅